src/main/java/com/alkemy/ong/repository/UserRepository.java

```diff
@@ -1,9 +1,10 @@
 package com.alkemy.ong.repository;
 
-
 import com.alkemy.ong.model.User;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface UserRepository extends JpaRepository<User, Long> {
+
+	User findByEmail(String email);
```

Comment:
Please wrap User in an Optional just in case it doesn't exists. Like: `Optional<User> findByEmail(String email);`